docs/modules/index.md

```diff
@@ -1,95 +1,92 @@
 
-### Install the Module
-To install a module, navigate to the Modules tab on the left-hand navigation of the admin dashboard of your Omeka S installation.
+You can also use SSH to clone a module directly from that module's git repository. Do this only if you are comfortable with git, GitHub, and working with SSH.
 
-Find the module in the table and click the install button.
-
-The module is now installed. You should see a success message once the page reloads. For modules with configuration options, you will be taken immediately to the configuration page for that module. For all other modules the Modules page will simply reload. On the Modules page, the *Status* of the module should change from “Not Installed” to “Not Active”
+### Install a module
+To install a module, navigate to the Modules tab on the left-hand navigation of the admin dashboard of your Omeka S installation.
 
-![Success message with green highlight reading “The module was successfully installed”](modulesfiles/mods_insuccess.png)
+Find the module in the table and click the "Install" button. The module is now installed and will automatically be activated. For modules with configuration options, you will be taken to the configuration page for that module. For modules without configuration options, the Modules page will simply reload, with a success message at the top.
 
-Modules are automatically activated when installed. 
+![Success message with green highlight reading "The module was successfully installed"](modulesfiles/mods_insuccess.png)
 
-### Activating Module
-If you have a module which is installed but inactive, click the activate button to use the module.
+### Activating a module
 
-You should see a success message once the page reloads, and the *Status* of the module should change from “Not Active” to “Active”
+If you have a module which is installed but inactive, click the "Activate" button to use the module. You should see a success message once the page reloads. The "Activate" button will now read "Deactivate".
 
-![Success message with green highlight reading “The module was successfully activated”](modulesfiles/mods_acsuccess.png)
+![Success message with green highlight reading "The module was successfully activated"](modulesfiles/mods_acsuccess.png)
 
 ## Configuring a module
-Some, but not all, modules include the option to configure settings for that module. If an active module displays the *configure* (wrench) button, you may configure that module.
+Some, but not all, modules include the option to configure settings for that module. If an active module displays the "Configure" button with a wrench icon, you should click through to understand the configuration settings for that module.
 
 Information regarding configuration options for individual modules can be found on the documentation page for those modules.
 
 ## Uninstalling a module
 Removing a module is essentially installing in reverse:
```

Comment:
We probably need a note here that uninstalling  module deletes its tables in the database and destroys any data that has been generated in conjunction with that module while it was installed.